`TestMember` failed

## Flaky Test

### Which jobs are failing
```
--- FAIL: TestMember (8.11s)
    member_test.go:98: 
        	Error Trace:	member_test.go:98
        	Error:      	"[ID:3809794510935671484 name:"pd1" peerURLs:"http://127.0.0.1:38411/" clientURLs:"http://127.0.0.1:43787/"  ID:12986232493755303981 name:"pd2" peerURLs:"http://127.0.0.1:40765/" clientURLs:"http://127.0.0.1:45041/"  ID:17740380411676238378 name:"pd3" peerURLs:"http://127.0.0.1:37813/" clientURLs:"http://127.0.0.1:40259/" ]" should have 2 item(s), but has 3
        	Test:       	TestMember
```

### CI link
https://github.com/tikv/pd/actions/runs/3458377971/jobs/5772744398

### Reason for failure (if possible)


### Anything else

--- a/tests/pdctl/member/member_test.go
+++ b/tests/pdctl/member/member_test.go
@@ -93,15 +93,19 @@ func TestMember(t *testing.T) {
 	args = []string{"-u", pdAddr, "member", "delete", "name", name}
 	_, err = pdctl.ExecuteCommand(cmd, args...)
 	re.NoError(err)
-	members, err = etcdutil.ListEtcdMembers(client)
-	re.NoError(err)
-	re.Len(members.Members, 2)
+	testutil.Eventually(re, func() bool {
+		members, err = etcdutil.ListEtcdMembers(client)
+		re.NoError(err)
+		return len(members.Members) == 2
+	})
 
 	// member delete id <member_id>
 	args = []string{"-u", pdAddr, "member", "delete", "id", fmt.Sprint(id)}
 	_, err = pdctl.ExecuteCommand(cmd, args...)
 	re.NoError(err)
-	members, err = etcdutil.ListEtcdMembers(client)
-	re.NoError(err)
-	re.Len(members.Members, 2)
+	testutil.Eventually(re, func() bool {
+		members, err = etcdutil.ListEtcdMembers(client)
+		re.NoError(err)
+		return len(members.Members) == 2
+	})
 }

